### DISCUSSION TITLE
How to I appeal a mistake?

### DISCUSSION THREAD

**Discussion by mushi284** [ValveSoftware/Dota2-Gameplay#17442] [Q&A]
Hello, I am a new player on Dota 2 and I started playing with friends a few days ago. One match today, I found that I suddenly couldn't pick a hero in an unranked game. I kept clicking and clicking, but it kept saying "I am recommending [hero]." At the time, I had no idea what the keybind was, nor that it even existed in game. So, the timer ran out and I was locked from matchmaking for a bit of time. Even though it seems like a trivial matter, I am quite spooked this happened and sad that I ruined a perfectly good match due to my inexperience. What can I do to fix this?

**Comment by commentator24** [comment]
#17390 alt bug
sticky key is stuck. turned on somehow, causing you as if you holding Alt and click same time (alt click is suggest hero).
you can restart dota or try Shift+Alt few times at same time to try unstuck it. i think unplug then plug keyboard might work too

usually happens when you quickly alt tab after accepting game or something while waiting other players to accept. so maybe dont alt tab
